src/helpers.js

```diff
@@ -1,5 +1,13 @@
 import { API_KEY } from 'react-native-dotenv';
 
+export const mana = {
```

Comment:
👍 for constants! Maybe separate the api calls and constants into their own files? `constants.js` `networking.js`?